build/linux/arch/PKGBUILD

```diff
@@ -0,0 +1,53 @@
+# Maintainer: Meriadec Pillet <meriadec.pillet@gmail.com>
+# shellcheck disable=SC2154,SC2034,SC2164
+
+pkgname=ledger-live
+pkgver=1.0.3
```

Comment:
what will be the process of updating this each time we do a release. manually?

Reply:
We update the `pkgver` manually then we run `makepkg -g` to generate the `md5sums=(...)` var, then we run `scripts/publish-arch-package.sh` to publish it.

This can be documented in confluence, and even automated.

Reply:
documented on confluence :ok_hand: